src/scicloj/02_numpy.clj

```diff
@@ -439,17 +858,59 @@ plt.ylabel('number'));
 ^k/dataset
 (ds/head csv-data 3)
 
+(tech.v3.datatype.functional/mean (dtype/->reader (-> csv-data last last)))
+
+(letfn [(mean [x] (float (/ (apply + x) (count x))))]
+  (nd-aggregate mean
+                (dtype/->reader (-> csv-data last last))))
+
+
 ["Computation on Arrays: Broadcasting
  ------------------------------------------------"]
```

Comment:
I guess we should mention `tech.v3.tensor/broadcast` and add some examples. (Contrary to NymPy, it does not require the smaller dimension to be 1 as long as the longer dimension is its multiple.)

Reply:
Yes -- good point.  I wasn't sure what the approach to broadcasting should be, so I left it out.